app/otr/client/rocketchat.otr.js

```diff
@@ -94,7 +96,8 @@ Meteor.startup(function () {
 				}
 				if (data.userId !== Meteor.userId()) {
 					return otrRoom.encryptText(ack).then((ack) => {
-						Meteor.call('updateOTRAck', message._id, ack);
+						message.msg = message.msg !== msg ? msg : message.msg;
```

Comment:
This whole `Meteor.startup()` call must be moved to an isolated module under `client/startup` for the sake of standardization.